File: sum-odd.cpp
#include <iostream>

using namespace std;

int main()
{
    int i, n, sum=0, rem;
    cout << "Enter a number: " << endl;
    cin >> n;
    for (i=1; i<=n; i++)
    {
        if (i%2!=0) {
            sum=sum+i;
        }
    }
    cout << sum;
    return 0;
}

File: python/sum-odd.py
'''Sum of odd numbers to 0 to n'''
def sum_of_odd_numbers(n):
    total = 0
    for i in range(1, n + 1, 2):
        total += i
    return total

num = int(input("Enter a number: "))
result = sum_of_odd_numbers(num)
print(f"The sum of odd numbers between 1 to {num} is: {result}")
